website/components/mdx/_getting-started-download-section.mdx

```diff
@@ -13,7 +13,7 @@ You can download the precompiled binary from [releases](https://github.com/cloud
 <Tab>
 
 ```bash
-curl -L https://github.com/cloudquery/cloudquery/releases/download/cli/v0.32.12/cloudquery_linux_x86_64 -o cloudquery
+curl -L https://github.com/cloudquery/cloudquery/releases/latest/download/cli/cloudquery_linux_x86_64 -o cloudquery
```

Comment:
We can't use `latest` because of the monorepo. We update the hard coded version in https://github.com/cloudquery/cloudquery/blob/217609b8504affb4c560a14a56b84e91f91fd426/.github/workflows/release_cli.yml#L79